todo_app/tests_e2e/test_e2e.py

```diff
@@ -31,17 +32,17 @@ def driver():
  
 @pytest.fixture(scope="module")
 def app_with_temp_board():
-    # Create the new board & update the board id environment variable
-    trello_board = TrelloBoard.create_board('Test Board')
-    os.environ['TRELLO_BOARD_ID'] = trello_board.board_id
+    os.environ['TODO_DB_NAME'] = 'test-todo'
+    todo_board = ToDoBoard('Test Board')
+    os.environ['TODO_BOARD_ID'] = todo_board.board_id
+    test_to_do_status = 'To Do'
+    os.environ['test_to_do_status'] = test_to_do_status
```

Comment:
Is there any reason you set this environment variable? I don't think you ever use it again. For the other status, you use it in the test but I'd be inclined to just make it a file constant rather than an environment variable